interop-testing/src/main/java/io/grpc/testing/integration/TestCases.java

```diff
@@ -26,9 +26,11 @@
   CACHEABLE_UNARY("cacheable unary rpc sent using GET"),
   LARGE_UNARY("single request and (large) response"),
   CLIENT_COMPRESSED_UNARY("client compressed unary request"),
+  CLIENT_COMPRESSED_UNARY_NOPROBE("client compressed unary request (skip initial feature-probing request)"),
```

Comment:
This line (and `CLIENT_COMPRESSED_STREAMING_NOPROBE`) exceeds 100 characters. Can you change them to the format:

```
  CLIENT_COMPRESSED_UNARY_NOPROBE(
      "client compressed unary request (skip initial feature-probing request)"),
```